scanner/image/scanner/download-mappings.sh

```diff
@@ -2,8 +2,23 @@
 
 set -euo pipefail
 
-output_dir="/mappings"
-mkdir $output_dir
+if [[ "$#" -lt "1" ]]; then
+  >&2 echo "Error: please pass target directory as a command line argument."
+  exit 1
+fi
 
-curl --retry 3 -sS --fail -o "${output_dir}/repository-to-cpe.json" https://access.redhat.com/security/data/metrics/repository-to-cpe.json
-curl --retry 3 -sS --fail -o "${output_dir}/container-name-repos-map.json" https://access.redhat.com/security/data/metrics/container-name-repos-map.json
+output_dir="$1"
+shift
+
+mkdir -p "$output_dir"
+
+urls=(
+    "https://access.redhat.com/security/data/metrics/repository-to-cpe.json"
+    "https://access.redhat.com/security/data/metrics/container-name-repos-map.json"
+)
+
+for url in "${urls[@]}"; do
+    filename=$(basename "$url")
+    echo "Downloading ${url} > ${output_dir}/$filename"
+    curl --retry 3 -sS --fail -o "${output_dir}/$filename" "$url"
+done
```

Comment:
[minor] I'd add a message in the end so that it's clear from logs that the script completed as expected.

```suggestion
done

echo "Done"
```